src/onecontainer_api/backends/dlrs-pytorch/classifier/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM sysstacks/dlrs-pytorch-ubuntu:latest
 
 WORKDIR /classifier/
 COPY requirements.txt .
-RUN pip install --no-cache-dir -U -r requirements.txt
+RUN pip install -U -r requirements.txt
```

Comment:
nit: --no-cache-dir

Reply:
The problem here was those changes were made on github repo but not on the internal repo. Let me fix them.